wowup-electron/main.ts

```diff
@@ -95,6 +97,13 @@ log.transports.file.resolvePath = (
 };
 log.info("Main starting");
 
+if (!preferenceStore.get(useHardwareAccelerationKey)) {
```

Comment:
can we make this if(preferenceStore.get(useHardwareAccelerationKey) === false.toString()) so it defaults to on?

Reply:
That makes sense!